src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,111 @@
+package duke.main;
+
+import duke.exceptions.DukeException;
+import duke.exceptions.UnknownCommandException;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Main application class for a To Do List Application
+ */
+public class Duke extends Application {
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+
+    /**
+     * Storage object used to store TaskList object into a Storage mechanism (e.g. File)
+     */
+    private Storage storage;
```

Comment:
I don't quite understand why both the Duke and TaskList objects maintain a reference to the Storage object. Perhaps only one reference to it is enough?